front/src/applications/editor/tools/trackSplit/components/TrackSplitLeftPanel.tsx

```diff
@@ -100,8 +100,10 @@ const TrackSplitLeftPanel: FC<unknown> = () => {
                 : undefined
             }
             onChange={(e) => {
-              if (!Number.isNaN(e.target.valueAsNumber))
-                setState((prev) => ({ ...prev, offset: e.target.valueAsNumber * 1000 }));
+              setState((prev) => ({
+                ...prev,
+                offset: (!Number.isNaN(e.target.valueAsNumber) ? e.target.valueAsNumber : 0) * 1000,
```

Comment:
```suggestion
                offset: e.target.valueAsNumber * 1000 || 0,
```

Reply:
!Number.isNaN is not a really good test, it says true for anything that is not `NaN`
![Screenshot_20240628_163719](https://github.com/OpenRailAssociation/osrd/assets/24652545/4a780d49-1e7f-46c6-8c3d-53760e383cc1)

Reply:
`offset: e.target.valueAsNumber * 1000` is going to be a number only if `e.target.valueAsNumber`

Reply:
Ok thanks for the explanations! Done